api-reference/openapi.json

```diff
@@ -161,6 +166,30 @@
       }
     },
     "schemas": {
+      "FollowupGenerationGuidanceBehavior": {
+        "type": "string",
+        "description": "Different ways Aftercare AI will generate follow-up questions based on non-applicable user input.\n\n* `Succinct` - The AI will not generate a follow-up question if the response is irrelevant or complete. This is the default behavior.\n* `Redirection` - The AI will return a follow-up question that redirects the user to answering the top-level question if it is irrelevant and incomplete.\n* `Verbose` - The AI will return a follow-up question regardless of the relevance or completeness of the response. This is the most verbose behavior.",
```

Comment:
This is what it looks like:
![Screenshot 2024-12-21 at 8 32 35 PM](https://github.com/user-attachments/assets/ce9b0111-df85-4adb-ad38-4c6c50c0a50a)


Seems like mintlify didn't support a better way than just putting the enum descriptions in the description.